### PR TITLE
Rename Swagger-spec.yaml to Open-api.yaml

### PR DESCRIPTION
**Pull request by kpanot**
Depending to #1744

### CODE REVIEW

File: packages/@ama-sdk/schematics/README.md [hunk 1]
```diff
@@ -242,6 +242,17 @@ yarn schematics @ama-sdk/schematics:typescript-core --generator-key example-sdk
...
+### Migration
+
+To help to apply changes on the Shell part of the SDK repository, a `migrate` schematic is exposed:
+
+```shell
+yarn schematics @ama-sdk/schematics:migrate --from 10.0.0
+```
+
+> [!NOTE]
+> The parameters `--from` is mandatory to provide the version of the `@ama-sdk/schematics` package original from which the rules should be run.
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/README.md:254]:
```suggestion
> The `--from` parameter is mandatory to provide the version of the original `@ama-sdk/schematics` package from which the rules should be run.
```

File: packages/@ama-sdk/schematics/schematics/migrate/schema.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "MigrateSchematicsSchema",
+  "title": "Execute migration scripts between 2 versions",
+  "description": "Schematics to migrate from a specific version to another one",
+  "properties": {
+    "from": {
+      "type": "string",
+      "description": "Starting version from which executing the migration scripts",
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/schematics/migrate/schema.json:9]:
```suggestion
      "description": "Starting version from which the migration scripts are executed",
```

File: packages/@ama-sdk/schematics/schematics/migrate/schema.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/** Schematic Option */
+export interface MigrateSchematicsSchemaOptions {
+  /** Starting version from which executing the migration scripts */
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/schematics/migrate/schema.ts:3]:
```suggestion
  /** Starting version from which the migration scripts are executed */
```

File: packages/@ama-sdk/schematics/schematics/migrate/schema.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/** Schematic Option */
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/schematics/migrate/schema.ts:1]:
```suggestion
/** Migrate Schematics Options */
```

File: packages/@ama-sdk/schematics/schematics/migrate/index.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    updateRegenScript
+  ]
+};
+
+const isTypescriptSdk = (tree: Tree) => {
+  return tree.exists('/tsconfig.json');
+};
+
+/**
+ * Generate a Extension of a API core definition
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/schematics/migrate/index.ts:19]:
```suggestion
 * Generate an Extension of an API core definition
```

File: packages/@ama-sdk/schematics/schematics/migrate/index.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  const to: string = options.to || JSON.parse(require(resolve(__dirname, '..', '..', 'package.json'))).version;
+
+  return (tree, context) => {
+    const runMigrateSchematic = isTypescriptSdk(tree) ? getMigrationRuleRunner(tsMigrationMap, { logger: context.logger }) : undefined;
+    return runMigrateSchematic?.({from: options.from, to});
+  };
+}
+
+/**
+ * Generate a Extension of a API core definition
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/schematics/migrate/index.ts:33]:
```suggestion
 * Generate an Extension of an API core definition
```

File: packages/@ama-sdk/schematics/migration.json [hunk 7]
```diff
@@ -5,6 +5,16 @@
       "version": "10.0.0-alpha.0",
       "description": "Updates of @ama-sdk/schematics to v10.0.*",
       "factory": "./schematics/ng-update/index#updateV10_0"
+    },
+    "migration-v10_1": {
+      "version": "10.1.0-alpha.0",
+      "description": "Updates of @ama-sdk/schematics to v10.1.*",
+      "factory": "./schematics/ng-update/index#updateV10_1"
+    },
+    "migration-v11_1": {
```
Comment (kpanot) [inline packages/@ama-sdk/schematics/migration.json:14]:
```suggestion
    "migration-v11_0": {
```

File: packages/@ama-sdk/schematics/README.md [hunk 8]
```diff
@@ -251,7 +251,8 @@ To help to apply changes on the Shell part of the SDK repository, a `migrate` sc
 yarn schematics @ama-sdk/schematics:migrate --from 10.0.0 [--to 11.0.0]
 ```
 
-> [!NOTE]
+> The `--from` parameter is mandatory to provide the version of the original `@ama-sdk/schematics` package from which the rules should be run.
```
Comment (mrednic-1A) [inline packages/@ama-sdk/schematics/README.md:254]: I think in the rebase this line has been doubled.

**Review by vscaiceanu-1a — APPROVED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Review by fpaul-1A — DISMISSED** [review]

File: packages/@ama-sdk/schematics/schematics/ng-update/typescript/v11.0/update-regen-script.ts [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+      return;
+    }
+
+    const swaggerFilePath = file.path.replace(/package\.json$/, 'swagger-spec.yaml');
+    if (!tree.exists(swaggerFilePath)) {
+      return;
+    }
+
+    packageJson.scripts[SCRIPT_REGEN_LABEL] = packageJson.scripts[SCRIPT_REGEN_LABEL]!.replace(/swagger-spec\.yaml/g, `${LOCAL_SPEC_FILENAME}.yaml`);
+    tree.overwrite(file.path, JSON.stringify(packageJson, null, 2));
```
Comment (cpaulve-1A) [inline packages/@ama-sdk/schematics/schematics/ng-update/typescript/v11.0/update-regen-script.ts:27]: we should also replace the swagger spec file from the openapitools.json

**Pull request closed without merging**